test/index.test.ts

```diff
@@ -3,49 +3,102 @@ import { LSP6Signer } from '../src/index';
 const signingKey =
   'ffeb17b9a6059fec3bbab63d76b060b7380cac7a62ce6621a134531a46458968';
 
+const signingAddress = '0x2b389f8EB52D16A105e02165a2AC1450461A237b';
+
+const testCases = [
+  'hello',
+  'horse',
+  'computer',
+  'week-end',
+  'amuse',
+  'breakfast',
+  'series',
+  'season',
+  'recovery',
+  'threshold',
+  'step',
+  'composer',
+  'mill',
+  'verdict',
```

Comment:
I think you can be more exotic in your test cases - add sentences, add hex strings, numbers, etc.